src/pages/PositionDetails/Contents.tsx

```diff
@@ -81,19 +82,6 @@ const CollateralWrapButton = styled(Button)`
   width: 80px;
 `
 
-const MoreLink = styled.a`
-  color: ${(props) => props.theme.colors.textColor};
-  cursor: pointer;
-  font-size: 14px;
-  font-weight: 600;
-  margin: 0 0 0 12px;
-  text-decoration: underline;
-
-  &:hover {
-    text-decoration: none;
-  }
-`
-
```

Comment:
The `MoreLink` is used in the in this file and throws an error

![Screenshot_20201028_094941](https://user-images.githubusercontent.com/1144028/97438058-10a86600-1903-11eb-9be8-539538376513.png)